circle.yml

```diff
@@ -16,7 +16,8 @@ dependencies:
     - "~/.electron"
 
   pre:
-    - brew install node || brew upgrade node
+    - brew update
+    - brew upgrade node@8
```

Comment:
So the steps here might be a bit weird, but here's what I believe is happening on`master` using [this build](https://circleci.com/gh/desktop/desktop/1230):

 1. `brew install node` executes, it encounters the Ruby error:

```
/usr/local/Homebrew/Library/Homebrew/brew.rb:12:in `<main>': Homebrew must be run under Ruby 2.3! You're running 2.0.0. (RuntimeError)
```

 2. Homebrew then goes and updates itself (because reasons?!?!?):

```
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (caskroom/cask).
```

 3. It continues on to `brew upgrade node`, which now completes:

```
==> Upgrading 1 outdated package, with result:
node 8.9.0
==> Upgrading node 
==> Installing dependencies for node: icu4c
==> Installing node dependency: icu4c
...
```

So I'm breaking this out to make each step a bit more explicit (and pin this to the `8.x` series)

Reply:
This is awesome!